appservice/src/intTest/groovy/nu/yona/server/AddDeviceTest.groovy

```diff
@@ -2,108 +2,113 @@ package nu.yona.server
 
 
 import groovy.json.*
-import spock.lang.Shared
 
-class AddDeviceTest extends AbstractAppServiceIntegrationTest {
-
-	@Shared
-	def richardQuin
-
-	def 'Add user Richard'(){
+class AddDeviceTest extends AbstractAppServiceIntegrationTest
+{
+	def 'Set new device request'()
```

Comment:
Setting the device request (from your old device) and getting it (from your new device) is functionally different! So I would like to have two tests for that or a test method that reflects that they are both tested.